src/modules/sync/sync/assetFolders.ts

```diff
@@ -0,0 +1,52 @@
+import { AssetFolderModels, ManagementClient } from "@kontent-ai/management-sdk";
+import { match } from "ts-pattern";
+
+import { logInfo, LogOptions } from "../../../log.js";
+import { throwError } from "../../../utils/error.js";
+import { apply } from "../../../utils/function.js";
+import { omit } from "../../../utils/object.js";
+import { DiffModel } from "../types/diffModel.js";
+import { getTargetCodename, PatchOperation } from "../types/patchOperation.js";
+
+export const syncAssetFolders = async (
+  client: ManagementClient,
+  operations: DiffModel["assetFolders"],
+  logOptions: LogOptions,
+) => {
+  logInfo(logOptions, "standard", "Updating asset folders");
+
+  if (!operations.length) {
```

Comment:
Other entities also print this message always (even when there are no changes). I wanted it to be consistent. But it also makes sense. At least the user knows that the tools updates the entity and since there are no specific logs after that... there are no updates to be applied.